learning_examples_py/migrations_cpswap/listen_logsubscribe.py

```
@@ -0,0 +1,131 @@
+                print(f"[INFO] Subscription response: {response}")
+
+                while True:
+                    try:
+                        response = await asyncio.wait_for(websocket.recv(), timeout=60)
+                        data = json.loads(response)
+                        log_data = data["params"]["result"]["value"]
+                        logs = log_data.get("logs", [])
+                        signature = log_data.get("signature", "unknown")
+
+                        is_migrated = any(
+                            "Program log: Instruction: MigrateToCpswap" == log
```

Ruff (0.12.2)

Yoda condition detected

Rewrite as `log == "Program log: Instruction: MigrateToCpswap"`

(SIM300)